web/src/locales/fr.po

```diff
@@ -1846,201 +1852,217 @@ msgstr ""
 
 #: src/edit/NeighbourhoodMode.svelte
 msgid "Editing tools"
-msgstr ""
+msgstr "Outils de modifications"
 
 #: src/edit/NeighbourhoodMode.svelte
 msgid "<0/> Show before edits"
-msgstr ""
+msgstr "<0/> Masquer les modifications"
+# TODO: here I slightly changed the meaning (hide modifications)
 
 #: src/edit/NeighbourhoodMode.svelte
 msgid "Add a modal filter (hotkey 1)"
-msgstr ""
+msgstr "Ajouter un filtre modal (raccourci clavier 1)"
 
 #: src/edit/NeighbourhoodMode.svelte
 msgid "Add a modal filter"
-msgstr ""
+msgstr "Ajouter un filtre modal"
 
 #: src/edit/NeighbourhoodMode.svelte
 msgid "Toggle one-way (hotkey 2)"
-msgstr ""
+msgstr "Définir un sens-unique (raccourci clavier 2)"
+# TODO: here I slightly changed the meaning (define a one way)
```

Comment:
This seems fine. "Toggle one-way" is kind of weird in english anyway -- it's more like "toggle between one-way forwards, one-way backwards, and two-ways"